### PR TITLE
Homework.lambda.part3.exercise2 (kovrov)

### PR DESCRIPTION
**Pull request by vkovrovEPAM**

### CODE REVIEW

File: src/test/java/lambda/part3/exercise/Exercise2.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+    @Test
+    void mapEmployeesToCodesOfLetterTheirPositionsUsingMapHelper() {
+        List<Employee> employees = getEmployees();
+
+        List<Integer> codes = MapHelper.from(employees)
+                .flatMap(Employee::getJobHistory)
+                .map(JobHistoryEntry::getPosition)
+                .flatMap(Exercise2::calcCodes)
+                .map(c -> c)
```
Comment (lesscau) [inline src/test/java/lambda/part3/exercise/Exercise2.java:81]: this operation seems redundant

Comment (vkovrovEPAM) [reply src/test/java/lambda/part3/exercise/Exercise2.java:81]: Yes, but it is a line in test wrote by @elefus

**Review by lesscau — APPROVED** [review]

File: src/test/java/lambda/part3/exercise/Exercise2.java (hunk 1 above)
Comment (AnyaYu) [reply src/test/java/lambda/part3/exercise/Exercise2.java:81]: According to the task flatMap(String -> Character(letter)) flatMap should return a List of Characters, not a List of Integers.

Comment (vkovrovEPAM) [reply src/test/java/lambda/part3/exercise/Exercise2.java:81]: @AnyaYu Discussed with @elefus , it is fine to do this way

**Review by AnyaYu — APPROVED** [review]

**Pull request merged**